services/course-material/src/components/ContentRenderer/moocfi/FlipCard/FlipCardBlock.tsx

```diff
@@ -0,0 +1,135 @@
+import { css } from "@emotion/css"
+import React, { useState } from "react"
+import { useTranslation } from "react-i18next"
+
+import ContentRenderer, { BlockRendererProps } from "../.."
+
+import { Block } from "@/services/backend"
+import withErrorBoundary from "@/shared-module/common/utils/withErrorBoundary"
+
+interface FlipCardAttributes {
+  backgroundColor: string
+  size: string
+}
+
+function isBlockImage(block: Block<unknown>): block is Block<FlipCardAttributes> {
+  return block.name === "core/image"
+}
+
+const FlipCardBlock: React.FC<React.PropsWithChildren<BlockRendererProps<FlipCardAttributes>>> = (
+  props,
+) => {
+  const { t } = useTranslation()
+  const frontCard = props.data.innerBlocks[0] as Block<FlipCardAttributes>
+  const backCard = props.data.innerBlocks[1] as Block<FlipCardAttributes>
+
```

Comment:
Check if frontCard and backcard are defined and return an error banner if they are not.